src/stan/lang/generator/generate_function_body.hpp

```diff
@@ -31,7 +31,8 @@ namespace stan {
       o << " {" << EOL;
       o << INDENT << "typedef " << scalar_t_name << " fun_scalar_t__;" << EOL;
       o << INDENT << "typedef "
-        << ((fun.return_type_.base_type_ == INT_T) ? "int" : "fun_scalar_t__")
+        << (fun.return_type_.base_type_.is_int_type() ?
+            "int" : "fun_scalar_t__")
```

Comment:
I strongly prefer operators to start lines, not end lines in continued expressions (it's also the convention in math typesetting).

[optional]

Reply:
fixed.